src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -150,7 +150,9 @@ export var resourceDropdownDirective = (
 export var modalActionDirective = () => {
     return {
         restrict: "E",
-        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"toggle();\">{{ label | translate }}</a>",
+        transclude: true,
+        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"toggle();\">" +
+            "<ng-transclude></ng-transclude>{{ label | translate }}</a>",
```

Comment:
I've done this using `ng-transclude` rather than `adh-inject`, for the sake of simplicity. I can change it if desired.